src/radical/saga/adaptors/lsf/lsfjob.py

```diff
@@ -40,9 +40,17 @@
 # FIXME: will be taken from resource config
 RESOURCES = {
     'summit': {'cpn': 42,
-               'gpn': 6},
+               'gpn': 6,
+               'valid_alloc_flags': [
```

Comment:
Maybe `valid_options`?

Reply:
that also works, but I have a point why did pick that one - since it is inside of LSF adaptor, here we put all values that are related to `alloc_flags` (the actual values provided by user/config are taken from `jd.system_architecture['options']`)

Reply:
Yeah, that's fair, works for me.  Thanks.